### PR TITLE
Created API contract for Discord Bot API

### LINKED ISSUE #106: Create API contract for Discord Bot API
## What to do?
- Create a APi contract for discord bot which adds the data whenever a `/link` or `/verify` command is being run.
- The request to the backend from Cloudflare worker (Discord Bot) has to be verified by a JWT token
- If the token is expired, the request is invalid
- If the payload is incorrect then also the request is invalid

### PR DESCRIPTION
**Pull request by rohan09-raj**
## Issue
Closes #106 

## Additions
- Created a APi contract for discord bot which adds the data whenever a `/link` or `/verify` command is being run.
- The request to the backend from Cloudflare worker (Discord Bot) is being verified by the JWT token
- If the token is expired, the request is invalid
- If the payload is incorrect then also the request is invalid

### CODE REVIEW

File: extension-requests/README.md [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
 
-|               Route                |    Description    |
-| :--------------------------------: | :---------------: |
-|      [GET /extension-requests](#get-extension-requests)      | Return all extension requests |
-|      [GET /extension-requests/:id](#get-extension-requests-id)      | Return extension request with id |
-|      [GET /extension-requests/self](#get-extension-requests-self)      | Return all extension requests of a user |
-|     [POST /extension-requests](#post-extension-requests)     | Creates new extension request  |
-|     [PATCH /extension-requests/:id](#patch-extension-requestsid) |   Updates extension request   |
-|     [PATCH /extension-requests/:id/status](#patch-extension-requests-id-status) |   APPROVE or DENY an extension request   |
+|                                    Route                                    |               Description               |
```
Comment (RitikJaiswal75) [inline extension-requests/README.md:19]: Why is this file changed

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

File: extension-requests/README.md (hunk 1 above)
Comment (rohan09-raj) [reply extension-requests/README.md]: Resolved
